qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/TestData.java

```diff
@@ -684,4 +678,59 @@ public String toString() {
 		public String datetime;
 		public String uuid;
 	}
+
+	public static class PatientGenerator {
+
+		private static final String[] PATIENT_GIVEN_NAMES = {"Alexandre", "Achint",
+				"Brittany", "Burke", "Cesar", "Cosmin", "Daniel", "Darius",
+				"David", "Ellen", "Émerson", "Evan", "Fernando", "Gabou",
+				"Glauber", "Hamish", "Louise", "Mário", "Mark", "Natália", "Neil",
+				"Neissi", "Nelice", "Rafal", "Renee", "Wyclif"};
+		private static final String[] PATIENT_FAMILY_NAMES = {"Barbosa", "Sethi",
+				"Eddy", "Mamlin", "Vortmann", "Ioan", "Kayiwa", "Jazayeri",
+				"Walton", "Ball", "Hernandez", "Waters", "Freire", "Mandy",
+				"Ramos", "Fraser", "Sécordel", "Areias", "Goodrich", "Arsand",
+				"Craven", "Lima", "Heck", "Korytkowski", "Orser", "Luyima"};
+		private static final String[] PATIENT_BIRTH_MONTH = MONTHS;
+		private static final String[] PATIENT_BIRTH_DAY = DAYS;
+		private static final String[] PATIENT_BIRTH_YEAR = YEARS;
+		private static final String[] PATIENT_GENDER = {"Male", "Female"};
+
+		public static org.openmrs.contrib.qaframework.helper.TestPatient generateTestPatient() {
+			org.openmrs.contrib.qaframework.helper.TestPatient testPatient = new org.openmrs.contrib.qaframework.helper.TestPatient();
```

Comment:
isn't it possible to have a better way of declaring an object of TestPatient?

Reply:
any suggestion @kdaud

Reply:
yes I think

Reply:
Since the issue is caused by having two classes having the same name, could we rename one of the class if it solves the issue?

Reply:
both are in TestData class

Reply:
@kdaud take a look (https://github.com/openmrs/openmrs-contrib-qaframework/pull/281#issuecomment-999326200)

Reply:
> Since the issue is caused by having two classes having the same name, could we rename one of the class if it solves the issue preferably the one not within the TestData!

removing one from Testdata class means we are changing the whole source code...

Reply:
The concept not removing TestData class. I think the reason of declaring the TestPatient object as 
`org.openmrs.contrib.qaframework.helper.TestPatient testPatient = new org.openmrs.contrib.qaframework.helper.TestPatient(); `

its because there are two class with the same name(TestPatient), my suggestion is to have one of the class renamed.

Reply:
Yah u are true @kdaud but this requires changing the whole code base ....isn't it better we create a ticket for it

Reply:
@Muta-Jonathan does this requires to just renaming one of the class? What happens when you do so?

Reply:
Have tried it locally but this goes beyond renaming `TestPatient class` ....it also extends to the page classes like the `RegistrationPage class` @kdaud

Reply:
>Renaming TestPatient class

![Screenshot from 2021-12-29 09-42-29](https://user-images.githubusercontent.com/86114366/147634611-ce81bea0-36aa-4a5f-b620-da19786a518e.png)



>Errors show up in classes

![Screenshot from 2021-12-29 09-44-00](https://user-images.githubusercontent.com/86114366/147634789-2f92d3fc-7849-4e92-98b8-312da32ac3eb.png)



![Screenshot from 2021-12-29 09-43-05](https://user-images.githubusercontent.com/86114366/147634711-6d93b515-0af6-46f5-83af-fc9d6a883946.png)

Reply:
would be better we create a ticket for it if important